pkg/sql/colexec/dispatch/dispatch.go

```diff
@@ -121,7 +122,7 @@ func (arg *Argument) waitRemoteRegsReady(proc *process.Process) (bool, error) {
 			return false, moerr.NewInternalErrorNoCtx("wait notify message timeout")
 		case <-proc.Ctx.Done():
 			arg.ctr.prepared = true
-			logutil.Infof("conctx done during dispatch")
+			logutil.Debugf("conctx done during dispatch")
```

Comment:
This error is probably a real error, like a network disconnection or something. I think it needs to be info